app/src/main/java/woowacourse/shopping/core/repository/CartRepository.kt

```
@@ -0,0 +1,23 @@
+import woowacourse.shopping.feature.cart.model.Cart
+
+object CartRepository {
+    private var cart: Cart = Cart()
+
+    fun getCart(): Cart = cart
+
+    fun addItem(product: Product): Boolean {
+        try {
+            cart = cart.addItem(product)
+            return true
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)